services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -3801,6 +3847,110 @@ paths:
+                  - data
+                properties:
+                  data:
+                    type: object
+                    required:
+                      - folders
+                    properties:
+                      folders:
+                        type: array
+                        items:
+                          $ref: '#/paths/~1folders/post/responses/200/content/application~1json/schema/properties/data'
+                  error:
+                    nullable: true
+                    default: null
+        default:
+          $ref: '#/components/responses/DefaultErrorResponse'
+    post:
+      tags:
+        - folder
+      summary: Creates a new folder
+      operationId: create_folder
+      responses:
+        '200':
+          description: The created folder
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - data
+                properties:
+                  data:
+                    type: object
+                    required:
+                      - name
+                      - color
+                    properties:
+                      id:
+                        type: integer
+                      name:
+                        type: string
+                      description:
+                        type: string
+                      color:
+                        type: string
+                        pattern: '^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$'
+                      created:
+                        type: string
+                        description: folder creation date
+                        pattern: '\d{4}-(12|11|10|0?[1-9])-(31|30|[0-2]?\d)T(2[0-3]|1\d|0?[0-9])(:(\d|[0-5]\d)){2}(\.\d{3})?Z'
+                        example: '2018-07-01T11:13:43Z'
+                      modified:
+                        type: string
+                        description: last change date
+                        pattern: '\d{4}-(12|11|10|0?[1-9])-(31|30|[0-2]?\d)T(2[0-3]|1\d|0?[0-9])(:(\d|[0-5]\d)){2}(\.\d{3})?Z'
+                        example: '2018-07-01T11:13:43Z'
+                  error:
+                    nullable: true
+                    default: null
+        default:
+          $ref: '#/components/responses/DefaultErrorResponse'
+  '/folders/{folder_id}':
+    parameters:
+      - name: folder_id
+        in: path
+        required: true
+        schema:
+          type: integer
+    put:
+      tags:
+        - folder
+      summary: Updates a folder
+      operationId: update_folder
+      responses:
+        '200':
+          description: The updated folder
+          content:
+            application/json:
+              schema:
+                $ref: '#/paths/~1folders/post/responses/200/content/application~1json/schema'
+        default:
+          $ref: '#/components/responses/DefaultErrorResponse'
+    delete:
+      tags:
+        - folder
+      summary: Deletes an existing folder
```

Comment:
This should only be possible if the folder has no associated linked projects, correct?